src/common/memory/MemoryTracker.h

```diff
@@ -11,6 +11,13 @@
 namespace nebula {
 namespace memory {
 
+constexpr size_t
+#if defined(__cpp_lib_hardware_interference_size)
+    L1_CACHE_LINE_SIZE = hardware_destructive_interference_size;
+#else
+    L1_CACHE_LINE_SIZE = 64;
```

Comment:
Minor：cache line size is same for all level of cpu caches,

Reply:
ACK.